src/Xamarin.Android.Tools.AndroidSdk/Sdks/AndroidSdkUnix.cs

```diff
@@ -94,6 +94,10 @@ protected override IEnumerable<string> GetAllAvailableAndroidSdks ()
 			if (!string.IsNullOrEmpty (preferedSdkPath))
 				yield return preferedSdkPath!;
 
+			foreach (string dir in GetSdkFromEnvironmentVariables ()) {
```

Comment:
This is "dead code", as `GetAllAvailableAndroidNdks()` is only called from `AndroidSdkBase.GetValidNdkPath()`.

Reply:
This is freshly added code and it's, eventually, called from `Initialize`

Reply:
Never mind. sorry!

I thought `FindBestNDK()` called `GetSdkFromEnvironmentVariables()` as well, and was mistaken.